test/unit/shimmer.test.js

```diff
@@ -106,10 +107,18 @@ describe('shimmer', function () {
         expect(counter).to.equal(1)
       })
 
+      it('should have some NR properties after instrumented', () => {
+        const mod = require(moduleName)
+        const nrKeys = getNRSymbols(mod)
+
+        const message = 'Expected some NR symbols but found none.'
+        expect(nrKeys.length, message).to.be.greaterThan(0)
```

Comment:
not a fan of this assertion. can we be explicit her as to which symbols you expect?

Reply:
Okay, sure, I explicitly am looking for the `'shim'` symbol now.